### PR TITLE
Add com.usebruno.Bruno

### PR DESCRIPTION
**Pull request by helloanoop**
See usebruno/bruno#623

Github: https://github.com/usebruno/bruno
Website: https://www.usebruno.com/

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

**Comment by helloanoop** [comment]
bot, build com.usebruno.Bruno

File: com.usebruno.Bruno.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - name: bruno
+    buildsystem: simple
+    sources:
+      - type: script
+        dest-filename: bruno
+        commands:
+          - zypak-wrapper.sh /app/extra/bruno/bruno --ozone-platform-hint=auto --enable-features=WaylandWindowDecorations "$@"
+
+      - type: extra-data
+        url: https://github.com/usebruno/bruno/releases/download/v1.5.1/bruno_1.5.1_x86_64_linux.AppImage
```
Comment (hfiguiere) [inline com.usebruno.Bruno.yml:33]: so how does this work on aarch64?

File: com.usebruno.Bruno.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+<component type="desktop-application">
+  <id>com.usebruno.Bruno</id>
+
+  <name>Bruno</name>
+  <summary>Fast and Git-Friendly Opensource API Client</summary>
+  <developer_name>Bruno</developer_name>
+  <url type="homepage">https://www.usebruno.com/</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
```
Comment (hfiguiere) [inline com.usebruno.Bruno.metainfo.xml:11]: the description say it's open source. which is it?

Comment (helloanoop) [reply com.usebruno.Bruno.metainfo.xml:11]: While majority of features are opensource, we plan to ship some paid offerings. Hence I kept the license as propietary.
Github: https://github.com/usebruno/bruno
Paid offering: https://www.usebruno.com/pricing

If needed, I can remove the `Opensource` word from the description

File: com.usebruno.Bruno.yml (hunk 1 above)
Comment (helloanoop) [reply com.usebruno.Bruno.yml:33]: As of now we want to support `x64`, we plan to add support for `aarch64` later.

Comment (hfiguiere) [reply com.usebruno.Bruno.yml:33]: you still build an aarch64 flatpak here.

Comment (helloanoop) [reply com.usebruno.Bruno.yml:33]: @hfiguiere Sorry, Can you expand. Are you saying the we should be able to add config for building `aarch64` in the future in the same manifest ?

Comment (bbhtt) [reply com.usebruno.Bruno.yml:33]: https://docs.flathub.org/docs/for-app-authors/maintenance#limiting-the-set-of-architectures-to-build-on

**Comment by helloanoop** [comment]
bot, build com.usebruno.Bruno

File: com.usebruno.Bruno.yml (hunk 1 above)
Comment (helloanoop) [reply com.usebruno.Bruno.yml:33]: Thanks @bbhtt 
I have added `flathub.json` file

**Comment by helloanoop** [comment]
I have tested the build and it's working!

**Comment by helloanoop** [comment]
@bbhtt @hfiguiere Can we get this merged ? 
Please let me know if there is anything else that I need to do.

Thanks.

**Comment by bbhtt** [comment]
Someone will get to it, please have patience. It's fine from my side.

**Comment by barthalion** [comment]
bot, build com.usebruno.Bruno

**Comment by helloanoop** [comment]
Thank you ❤️

**Pull request closed without merging**